src/platform/mhv/secondary-nav/tests/MhvSecondaryNavMenu.unit.spec.jsx

```diff
@@ -116,9 +116,18 @@ describe('MHV Secondary Navigation Menu Component', () => {
   });
 
   describe('renders', () => {
+    it('when window location pathname is empty', () => {
+      setWindowUrl('');
+      expect(getOneLink(medNavItem).className).to.not.include(
+        activeClassString,
+      );
+    });
+
     it('when window location pathname is not set', () => {
       delete window.location;
-      expect(getOneLink(medNavItem)).to.exist;
+      expect(getOneLink(medNavItem).className).to.not.include(
+        activeClassString,
+      );
```

Comment:
I'm wondering it makes the specs more clear if we add specific examples of when the activeStringClass is not applied. e.g. - `/my-health/update-benefits-information-form-10-10ezr`, `/my-health/order-medical-supplies`